nat.tf

```diff
@@ -1,10 +1,21 @@
+module "nat_label" {
+  source     = "git::https://github.com/cloudposse/terraform-null-label.git?ref=tags/0.2.2"
+  namespace  = "${var.namespace}"
+  stage      = "${var.stage}"
+  name       = "${var.name}"
+  delimiter  = "${var.delimiter}"
+  attributes = ["private"]
+  tags       = "${var.tags}"
+}
+
 locals {
   nat_gateways_count = "${var.nat_gateway_enabled == "true" ? length(var.availability_zones) : 0}"
 }
 
 resource "aws_eip" "default" {
   count = "${local.nat_gateways_count}"
   vpc   = true
+  tags  = "${module.private_label.tags}"
```

Comment:
did you mean using `nat_label` here

Reply:
@dcowan-vestmark did you see this comment?